app/Main.hs

```diff
@@ -1,6 +1,17 @@
 module Main where
 
-import Lib
+import Control.Monad (join)
+
```

Comment:
Usually I'll put all external imports (i.e. stuff from the cabal file that I didn't write) in one group, and local imports (stuff I did write) in another group. Given that, I'd put Control.Monad and Options.Applicative together.